src/components/Alert/Alert.vue

```diff
@@ -1,51 +1,122 @@
 <template>
   <div
-    :class="['w-full border-l-4 py-3 px-4 rounded-r-lg flex justify-start items-start text-sm',
-             { 'bg-turquoise-100 border-turquoise-500': info },
-             { 'bg-mint-100 border-success': success },
-             { 'bg-lemon-100 border-warning': warning },
-             { 'bg-coral-100 border-coral-700': error }
+    :class="['w-full py-3 px-4',
+             { 'bg-blue-50 text-blue-700': info },
+             { 'bg-green-50 text-green-700': success },
+             { 'bg-orange-50 text-orange-600': warning },
+             { 'bg-red-50 text-red-700': error }
     ]"
   >
-    <component
-      :is="icon"
-      class="h-6 w-6 flex-shrink-0"
-      :class="[{ 'text-gray-500': success },
-               { 'text-turquoise-500': info },
-               { 'text-warning': warning },
-               { 'text-coral-700': error }
-      ]"
-    />
-    <div class="px-4">
-      <slot :link-color="linkColor" />
+    <div
+      v-if="hasHeading"
+      class="flex justify-between"
+    >
+      <div class="flex">
+        <component
+          :is="icon"
+          v-if="showIcon"
+          class="h-6 w-6 flex-shrink-0 mr-2"
+          :class="[{ 'text-green-700': success },
+                   { 'text-blue-700': info },
+                   { 'text-orange-600': warning },
+                   { 'text-red-700': error }
+          ]"
+        />
+        <div class="type-base-700">
+          <slot name="heading" />
+        </div>
+      </div>
+      <LobLink
+        v-if="learnMoreLink"
+        :to="learnMoreLink"
+        :underline="false"
+        class="ml-4 !type-xs-400 text-gray-500"
+      >
+        {{ linkDisplayText }}
+        <ArrowRight class="inline w-3 h-3" />
+      </LobLink>
+      <button v-if="showCloseButton">
+        <Close class="w-4 h-4 ml-4" />
+      </button>
+    </div>
+    <div
+      v-if="hasContent"
+      class="flex items-center justify-between"
+    >
+      <div class="flex items-center">
+        <component
+          :is="icon"
+          v-if="showIcon && !hasHeading"
+          class="h-6 w-6 flex-shrink-0 mr-2"
+          :class="[{ 'text-green-700': success },
+                   { 'text-blue-700': info },
+                   { 'text-orange-600': warning },
+                   { 'text-red-700': error }
```

Comment:
just a suggestion, wdyt about having a computed prop with the text color, so we don't repeat this code 3 times?
So:
```
textColor () {
    return {
      'text-green-700': success,
      'text-blue-700': info,
      'text-orange-600': warning,
       'text-red-700': error
    }
  }
```
Could also have this for the background color